src/globus_cli/commands/group/member/add.py

```diff
@@ -0,0 +1,45 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import IdentityType, ParsedIdentity, command
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.types import FIELD_LIST_T
+
+
+def _format_added_user_id(x: dict):
+    return x["add"][0]["identity_id"]
+
+
+def _format_added_user_username(x: dict):
+    return x["add"][0]["username"]
+
+
+def _format_group_id(x: dict):
+    return x["add"][0]["group_id"]
+
+
+ADD_USER_FIELDS: FIELD_LIST_T = [
+    ("Group ID", _format_group_id),
+    ("Added User ID", _format_added_user_id),
+    ("Added User Username", _format_added_user_username),
+]
+
+
+@command("add", short_help="Add one or more members to a group")
+@click.argument("group", type=str)
+@click.argument("user", type=IdentityType())
+@LoginManager.requires_login(LoginManager.GROUPS_RS)
+def member_add(group: str, user: ParsedIdentity, login_manager):
+    """Add one or more members to a group"""
```

Comment:
Let's expand the helptext for this command a little bit to explain that `USER` can be a username or identity ID.
Unlike `group` -> `group_id`, I don't think there's a subtle indicator we can do here. I'd just explain in the help, similar to what we have for `globus session update`: https://github.com/globus/globus-cli/blob/8f9267e66159ed5816c3b2db1af7731c6665190f/src/globus_cli/commands/session/update.py#L89